install/kubernetes/Makefile

```diff
@@ -41,7 +41,7 @@ EXPERIMENTAL_OPTIONS := \
     --set bandwidthManager=true \
     --set operator.replicas=1
 
-USE_DIGESTS ?= true
+USE_DIGESTS ?= $(shell if grep -q '""' Makefile.digests; then echo "false"; else echo "true"; fi)
```

Comment:
I'm actually not 100% sure if this is what we intend for the v1.8/v1.9 branches, maybe this is just supposed to be `USE_DIGESTS ?= false`. Then anyone on those releases can opt in for newer helm releases.

Reply:
you meant "newer image digests"? If not, then I didn't understand what you were trying to explain here.